### ISSUE TITLE
Filter config file is expected in working dir instead of project's dir/src dir

### ISSUE THREAD

**Issue by damned1** [dotnet/docfx#1321]
### Title
If the filter config file name is provided without a full path DocFx is looking for this file in the working directory instead of the project's directory.

### Functional inpact
The filter config file cannot be specified per project.

### Repro steps
My metadata section of ```docfx.json``` looks as follow:
```
  "metadata": [
    {
      "src": [
        {
          "src": "../",
          "files": [
            "Client/*/**.csproj"
          ],
          "exclude": [
            "**/obj/**",
            "**/bin/**",
            "_site/**"
          ]
        }
      ],
      "dest": "api",
      "filter": "metadataFilter.yml"
    }
  ],
```

The project is in ```PathA``` directory. The ```metadaFilter.yml``` file is also in ```PathA``` directory.

When I perform the following command from ```PathB``` which is a location of docfx executable:
```PathB> docfx "PathA\docfx.json" --serve```

I get the ```FileNotFoundException``` with the following message:
> Filter Config file PathB/metadataFilter.yml does not exist!

### Expected result
I would expect the filter config file is searched in project's location which is ```PathA```.

### Actual result
Docfx is looking in ```PathB``` for the filter config file.

### Further topics
The open question here is does the location of the filter config file should apply the "src" key value which for example in my case is ".../" and the final path would be ```PathA/.../metadataFilter.yml```

**Comment by vwxyzh** [comment]
It will release in docfx 2.13 (about March 6).

**Comment by jskeet** [comment]
Suggest this issue can be closed now, given that 2.13 has been released.
